devito/ir/iet/nodes.py

```diff
@@ -288,13 +288,16 @@ def is_tensor(self):
         return not self.is_scalar
 
     @property
-    def is_scalar_assign(self):
-        """True if a scalar, non-increment expression."""
-        return self.is_scalar and not self.is_Increment
+    def is_definition(self):
+        """
+        True if it is an assignment, False otherwise
+        """
+        return ((self.is_scalar and not self.is_Increment) or
+                (self.is_tensor and isinstance(self.expr.rhs, ListInitializer)))
```

Comment:
`is_deifinition` is a bit of an odd name to me, why not `is_assign`?

Reply:
See previous discussion with @fablup